### PR TITLE
Preserve `package.json` indentation

### LINKED ISSUE #7: Maintain package.json indentation
Original issue: [avajs/ava](https://github.com/avajs/ava/issues/1358).

### PR DESCRIPTION
**Pull request by zkochan**
Close #7

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 'use strict';
+const fs = require('fs');
+const path = require('path');
 const test = require('ava');
 const tempfile = require('tempfile');
 const readPkg = require('read-pkg');
+const writeJsonFile = require('write-json-file');
 const m = require('.');
```
Comment (kevva) [inline test.js]: Might as well rewrite these to use `import` syntax.

Comment (zkochan) [inline test.js]: Is it a blocker? I can fix it tonight but I'd really need this published by tomorrow. I want to publish pnpm v1, with indentation detection

Comment (sindresorhus) [inline test.js]: Nah, that's outside the scope of this PR anyways.

**Comment by sindresorhus** [comment]
Published as 3.1.0

**Pull request merged**